Index: libmpack-1.0.3/binding/lua/lmpack.c
===================================================================
--- libmpack-1.0.3.orig/binding/lua/lmpack.c
+++ libmpack-1.0.3/binding/lua/lmpack.c
@@ -31,8 +31,10 @@
 
 #if LUA_VERSION_NUM > 501
 typedef luaL_Reg luaL_reg;
-#define luaL_register(L, name, lreg) (luaL_setfuncs((L), (lreg), 0))
-#define lua_objlen(L, idx) (lua_rawlen(L, (idx)))
+# if !defined(LUA_COMPAT_5_1)
+#  define luaL_register(L, name, lreg) (luaL_setfuncs((L), (lreg), 0))
+#  define lua_objlen(L, idx) (lua_rawlen(L, (idx)))
+# endif
 #endif
 
 typedef struct {
